api/src/main/java/jakarta/servlet/http/Cookie.java

```diff
@@ -92,6 +92,7 @@
     private boolean secure; // ;Secure ... e.g. use SSL
     private int version = 0; // ;Version=1 ... means RFC 2109++ style
     private boolean isHttpOnly = false;
+    private SameSite sameSite = null;
```

Comment:
Considering https://www.chromestatus.com/feature/5088147346030592, I wonder if this default should be `Lax`? 
Alternately the default should be a value that has the semantic of "get the containers default value for Same-Site", so the container can introduce the attribute to existing applications.  If an application really does not want Same-Site set, then they will need to either set the container default to null or the individual cookie default to null.

Reply:
I would not depend heavily on Chrome's behaviour for too long. Chances are they will change the default to Strict on a whim. Having a container default could work better.

Reply:
> Alternately the default should be a value that has the semantic of "get the containers default value for Same-Site", so the container can introduce the attribute to existing applications. If an application really does not want Same-Site set, then they will need to either set the container default to null or the individual cookie default to null.

Having a container or application or server or product scoped default is fine, but the spec
will need to specify exactly what sorts of non-specified configuration options are allowed
or required.

Also, be careful about changing the actual default because if it changes the behavior of
existing applications then you've introduced a backwards compatibility breakage which
at the very least needs to be clearly documented.  And no, it's no fair changing the
default based on the version of the deployment descriptor being used.

Reply:
I think the default for the cookie should be to use the container default.  The default for the container should be null - ie do nothing different than now.  But at least having a container default means that existing applications can be configured without code changes to not be ignored by Chrome (and to be more secure! double bonus).    See more comments below...